### PR TITLE
Moved FanisGk and prmurphy to inactive #569

### PR DESCRIPTION
**Pull request by kylemathias**
Just moved them to the bottom. I don't really know what order this is in. Oldest leave date to newest?

### CODE REVIEW

**Comment by kylemathias** [comment]
Can take a look at it here https://kylemathias.github.io/#!pages/team.md

**Comment by waryhermit** [comment]
@kylemathias It takes at least two approvals to get a PR merged. I think you will easily get two approvals on open-learning-exchange/open-learning-exchange.github.io#570 if you do it from a branch and not master. As it is, @dogi will have to decide if its enough of a problem or if its OK in this case.

**Comment by kylemathias** [comment]
I will wait for @dogi to look and decide. If this gets closed let me know and i will Reopen this issue, branch correctly and apply this fix for the issue.

File: pages/team.md [hunk 1]
```diff
@@ -35,3 +33,5 @@
 |[epistulae](epistulae.md)|   |   |
 |[pranesh-s](pranesh-s.md)| 20160525 | 20160603 |
 |[314159265359](314159265359.md)| 20160523 | 20160603 |
+|[FanisGk](FanisGk.md)| 20160801 | 20170131 |
```
Comment (dogi) [inline pages/team.md]: they should go to the top please ;)

**Comment by dogi** [comment]
@kylemathias does that mean you will redo the PR?

**Comment by kylemathias** [comment]
@dogi yes, i am going to branch on my local machine and make the changes there. Im looking in how to do that right now

**Pull request closed without merging**